### PR TITLE
add modify-vm-template task stub for ci

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
add modify-vm-template task stub for ci

**Special notes for your reviewer**:

**Release note**:

```
NONE
```
Signed-off-by: Karel Šimon <ksimon@redhat.com>

### CODE REVIEW

File: configs/modify-vm-template-task.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+task_name: modify-vm-template-task
+task_category: modify-vm-template-task
+main_image: quay.io/kubevirt/tekton-task-modify-vm-template-task:v0.0.1
```
Comment (atiratree) [inline configs/modify-vm-template-task.yaml:3]: missing the okd part..

Comment (ksimon1) [reply configs/modify-vm-template-task.yaml:3]: good catch, fixed

File: configs/modify-vm-template.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+task_name: modify-vm-template
+task_category: modify-vm-template
+main_image: quay.io/kubevirt/tekton-modify-vm-template:v0.0.1
```
Comment (atiratree) [inline configs/modify-vm-template.yaml:3]: I think this should have a `tekton-task` prefix as the other tasks. 

Are you using the onboard-new-task script?

**Pull request merged**